converter/converters/koki_service_to_kube_v1_service.go

```diff
@@ -0,0 +1,196 @@
+package converters
+
+import (
+	"fmt"
+	"reflect"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/koki/short/types"
+	"github.com/koki/short/util"
+)
+
+func Convert_Koki_Service_To_Kube_v1_Service(service *types.ServiceWrapper) (*v1.Service, error) {
+	var err error
+	kubeService := &v1.Service{}
+	kokiService := &service.Service
+
+	kubeService.Name = kokiService.Name
+	kubeService.Namespace = kokiService.Namespace
+	kubeService.APIVersion = kokiService.Version
+	kubeService.Kind = "Service"
+	kubeService.ClusterName = kokiService.Cluster
+	kubeService.Labels = kokiService.Labels
+	kubeService.Annotations = kokiService.Annotations
+
+	if len(kokiService.ExternalName) > 0 {
+		kubeService.Spec.Type = v1.ServiceTypeExternalName
+		kubeService.Spec.ExternalName = kokiService.ExternalName
+		return kubeService, nil
+	}
+
+	kubeService.Spec.Type = v1.ServiceTypeClusterIP
```

Comment:
What about `NodePort` and `LoadBalancer` services?

Reply:
It's set to NodePort later if it finds any NodePorts in the ServicePorts.
Then if there's LoadBalancer info, it'll upgrade the type to LoadBalancer.

I'll add comments explaining the process.